experimental/framework/tree-react-api/src/test/useTree.spec.ts

```diff
@@ -72,11 +71,10 @@ describe("useTree()", () => {
 
 	it("works", () => {
 		const tree = createLocalTree("tree");
-		useTreeContext(tree.context);
-		assert.deepEqual(JSON.parse(JSON.stringify(tree.content)), {
+		// TODO test use functions
+		assert.deepEqual(JSON.parse(JSON.stringify(tree)), {
```

Comment:
I think this should work now, if you wanted to update it here or in a later PR:

```suggestion
		assert.deepEqual(tree), {
```

Reply:
Deep equal compares lots of things we don't want to compare, like object tags and prototypes. I think this is safer (at least less likely to break with nod version changes or implementation changes). If the object literal I created explicitly had null prototypes, maybe skipping the json round trip would make more sense.

On a related note, this also approach to comparing also fails to accurately compare polymorphic trees.